src/Service.Host/client/src/components/productRanges/ProductRange.js

```diff
@@ -0,0 +1,195 @@
+import React, { Component } from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import moment from 'moment';
+import { Grid } from '@material-ui/core';
+import {
+    InputField,
+    Title,
+    ErrorCard,
+    Loading,
+    SaveBackCancelButtons
+} from '@linn-it/linn-form-components-library';
+import PropTypes from 'prop-types';
+import Page from '../../containers/Page';
+
+const styles = () => ({});
+
+class ProductRange extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            productRange: props.productRange
+        };
+        this.handleFieldChange = this.handleFieldChange.bind(this);
```

Comment:
Is this bind needed?

Reply:
i think so but should die if we move away from components